Changelog.md

```diff
@@ -21,6 +21,7 @@ Compiler Features:
 
 Bugfixes:
  * Code Generator: Fix assertion failure when assigning structs containing array of mapping.
+ * Framework: Reset the Yul string repository before each compilation, freeing up memory.
```

Comment:
YulOptimizer?

Reply:
It also affects inline assembly, ABIEncoderV2 and other things.